### DISCUSSION TITLE
Home Assistant OS 11.2.rc1

### DISCUSSION THREAD

**Discussion by agners** [home-assistant/operating-system#2940] [Announcements]
## Changes

* Bump rpi-rf-mod dt overlay to latest version (#2921) @jens-maus
* Bump buildroot to v2023.02.7 (#2936) @sairon
* Bump docker/build-push-action from 5.0.0 to 5.1.0 (#2935) @dependabot
* Bump actions/github-script from 6 to 7 (#2934) @dependabot
* fix: disable uas for wavlink adaptor (#2924) @bertrandgressier
* Bump dessant/label-actions from 3.1.0 to 4.0.0 (#2922) @dependabot
* Bump buildroot to add RTL885x firmware (#2911) @sairon
* Fix incorrect uppercase Y kconfig use (#2908) @jens-maus
* Refresh all boards' defconfigs (#2895) @sairon
* generic-x86-64: Add missing dependency for #2800 to kernel.config (#2813) @xtsrthm
* Maintain and upload artifacts index (#2839) @agners

## Home Assistant Operating System

* Use kernel local version for HAOS compiled Linux kernel (#2912) @agners
* Improve Bluetooth cache cleanup command (#2906) @agners
* Bump buildroot to update Docker to v24.0.7 (#2883) @agners

## Build System

* Revert "Upload artifacts index even on partial builds (#2901)" (#2905) @agners
* Upload artifacts index even on partial builds (#2901) @agners
* Add pruning of object file cache (#2899) @sairon
* Fix Artifacts index bump when using workflow_call (#2900) @agners
* Pass secrets to artifacts index generation workflow (#2893) @agners
* Improve handling of timeouts in tests (#2890) @sairon
* Checkout source code when uploading artifacts index (#2892) @agners
* Fix the condition for OS artifact download (#2891) @sairon
* Fix quotes in artifact index generation (#2889) @agners
* Separate artifacts index update into separate workflow (#2888) @agners
* Add a flag to determine tests were called from another workflow (#2887) @sairon
* Bump HAOS beta when stable version is released (#2885) @sairon
* Always use environment for bump_version job (#2884) @sairon
* Add test suite for Supervisor tests (#2880) @sairon
* Run tests within build workflow (#2882) @sairon
* Use the correct secret to clear cache on the Cloudflare frontend (#2881) @agners
* Print object cache statistics before upload (#2874) @agners
* Install AWS CLI tools in final build step (#2873) @agners
* Optimize build cache for dev builds (#2872) @sairon
* Add checkout step to bump_version job in the build workflow (#2871) @sairon
* Fix GitHub action build.yaml (#2868) @agners

## Raspberry Pi

* Bump buildroot to fix rpi-firmware hash (#2930) @sairon
* Bump Raspberry Pi firmware to latest stable version (#2929) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon
* RaspberryPi: Update kernel 6.1.58 - stable_20231030 (#2904) @sairon
* RaspberryPi: Update kernel 6.1.58 - stable_20231024 (#2864) @sairon

## Home Assistant Yellow

* Fix miniUART on Yellow with kernel HCI init enabled (#2931) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon
* RaspberryPi: Update kernel 6.1.58 - stable_20231030 (#2904) @sairon
* RaspberryPi: Update kernel 6.1.58 - stable_20231024 (#2864) @sairon

## Open Virtual Appliance

* Linux: Update kernel 6.1.63 (#2937) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon

## Generic x86-64

* Linux: Update kernel 6.1.63 (#2937) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon

## Hardkernel ODROID

* Linux: Update kernel 6.1.63 (#2937) @sairon
* Reduce Tinker and Amlogic kernel configs to defconfigs (#2925) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon

## ASUS Tinker

* Linux: Update kernel 6.1.63 (#2937) @sairon
* Reduce Tinker and Amlogic kernel configs to defconfigs (#2925) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon

## Khadas VIM Series

* Linux: Update kernel 6.1.63 (#2937) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon

## Generic aarch64

* Linux: Update kernel 6.1.63 (#2937) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon
* Generic AArch64: Enable options required for virtiofs (#2909) (#2910) @agners

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/11.2.rc1'>Home Assistant OS 11.2.rc1</a>.</em>

**Comment by bschatzow** [comment]
No issues with 11.2 rc1.  It looks like what ever was causing my reboot issue with my SSD and rpi4 is gone. Have not seen a problem on booting since early 10. releases

**Comment by mpadinhabrandao** [comment]
Hey 🖖 
Version 11.2 will support a Rpi5?

**Reply by sairon** [reply]
No, Raspberry Pi 5 support is still being worked on and is expected to arrive later, probably in HAOS 12 which doesn't have any ETA yet.

**Reply by mpadinhabrandao** [reply]
ok, I'll continue to wait.
Good job

**Comment by tjorim** [comment]
I'm not sure why, but as of OS 11.2.rc1 I have a bluetooth adapter showing up.
Could be related to supervisor but I don't notice any changes related to that.
It discovers hci0 but never inits on my HA Yellow (as my CM4 does not have wireless).
I don't have a big understanding of how the OS works but I assume it might be related to #2931 by @sairon.

Host logs:
```
Nov 23 21:35:19 homeassistant kernel: Bluetooth: hci0: command 0xfc18 tx timeout
Nov 23 21:35:27 homeassistant kernel: Bluetooth: hci0: BCM: failed to write update baudrate (-110)
Nov 23 21:35:27 homeassistant kernel: Bluetooth: hci0: Failed to set baudrate
Nov 23 21:35:29 homeassistant kernel: Bluetooth: hci0: command 0x0c03 tx timeout
Nov 23 21:35:37 homeassistant kernel: Bluetooth: hci0: BCM: Reset failed (-110)
...
```

Core:
```
Logger: bluetooth_auto_recovery.recover
Source: runner.py:188
First occurred: 22:35:37 (5 occurrences)
Last logged: 22:38:24

Bluetooth management socket connection lost: [Errno 22] Invalid argument
```

```
Logger: bluetooth_auto_recovery.recover
Source: components/bluetooth/util.py:78
First occurred: 22:35:16 (21 occurrences)
Last logged: 22:38:40

Could not determine the power state of the Bluetooth adapter hci0 [00:00:00:00:00:00] due to timeout after 5 seconds
Could not cycle the Bluetooth adapter hci0 [00:00:00:00:00:00]: [Errno 110] Operation timed out
Bluetooth adapter hci0 [00:00:00:00:00:00] could not be reset:
Closing Bluetooth adapter hci0 [00:00:00:00:00:00] failed: [Errno 9] Bad file descriptor
```
